README.md

```diff
@@ -1,59 +1,67 @@
+
+This particular application is built using Angular and uses the npm package `@laserfiche/lf-ui-components`, which can only be used in an Angular project. If you would like to see examples of how to use the Laserfiche UI Components in any framework (Angular, Vue, React, no-framework, etc.), please see our other sample projects in [Angular](https://github.com/Laserfiche/lf-sample-OAuth-SPA-angular-cdn) and [React](https://github.com/Laserfiche/lf-sample-OAuth-SPA-react).
+
+
+This project depends on the following libraries
+
+- From NPM:
+  - [@laserfiche/lf-js-utils](https://www.npmjs.com/package/@laserfiche/lf-js-utils)
+  - [@laserfiche/lf-ui-components-services](https://www.npmjs.com/package/@laserfiche/lf-ui-components-services)
+  - [@laserfiche/lf-ui-components](https://www.npmjs.com/package/@laserfiche/lf-ui-components)
+
+## Contribution
+
+We welcome contributions and feedback. Please follow our [contributing guidelines](./CONTRIBUTING.md).
+
+## Pre-Requisites
+
+- Visual Studio Code
+- Node 16 (LTS)
+- all other Angular pre-reqs are in the local package.json
+
+## First Time Setup
+
+0. Clone the repo
+1. Create a new Single Page App in devconsole for [US](https://app.laserfiche.com/devconsole/apps), [Canada](https://app.laserfiche.ca/devconsole/apps), or [Europe](https://app.eu.laserfiche.com/devconsole/apps) (depending on your region). This can also be accessed via the app picker in the top right when logged into your Laserfiche account.
+    - Add the authentication redirect URI (e.g.: <https://myapp.example.com/lf-sample-app/>). This redirect URI in the devconsole must match the REDIRECT_URI variable in `src/app/config.js`, which must match the address on which you are hosting your application.
+       - **Note: If you want to use the popup login as well, you must register the login page too (e.g. <https://myapp.example.com/lf-sample-app/popup-login.html>)
+   - Select required scope(s) needed to read and write to the repository in the 'Authentication' tab  ("repository.Read" and "repository.Write" ). Scopes are case-sensitive and space-delimited. 
+   - Note that in order to access the devconsole, your user account must have been given Developer Console Administrator Access Rights by an Account Administrator. Otherwise you will see a message along the lines of "You do not have permission to view the developer console."
+2. Open folder (containing package.json) with Visual Studio Code
+3. In `src/app/config.js` update the REDIRECT_URI and CLIENT_ID with the app client_id and redirect_uri as registered in step 1. You only need to update HOST_NAME if you are not using cloud production (laserfiche.com).
+4. In VS Code, open a New Terminal window.
+5. Run `npm install` in root folder. This command installs dependencies packages.
+6. Run `ng build` in root folder. This command builds the project into `dist/` directory. See below for suggestions on how to host this folder.
+
+## How to use `<lf-login>` component in your Sample app 

```

Comment:
change the title to something like: how to use <lf-login> component in a Popup Window
Because all the points below are referring to popup windows specifically